File: C++/Bit_manipulation/Get_Bit.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int n = 5;
    int position = 2;
    int  bitmask = 1<<position;

    if((n & bitmask) == 0){
        cout<<"Bit is Zero"<<endl;
    }else{
        cout<<"Bit is One"<<endl;
    }
}

File: JAVA/Bit_Manipulation/Get_Bit.java
import java.util.*;

//Get third 3rd bit of a number n
// Answer = 1. Bit mask : -   1 << i  2. operation :- AND

public class Get_Bit {
    public static void main(String[] args) {
        int n = 5;
        int pos = 2;
        int bitmask = 1<<pos;
        if((bitmask & n) == 0){
            System.out.println("Bit was Zero ");
        }else{
            System.out.println("Bit was One");
        }

    }
}
